pkg/runcfanotify/runcfanotify.go

```diff
@@ -413,16 +413,22 @@ func (n *RuncNotifier) monitorRuncInstance(bundleDir string, pidFile string) err
 
 	// watchPidFileIterate() will read config.json and it might be in the
 	// same directory as the pid file. To avoid getting events unrelated to
-	// the pidfile, add an ignore mask.
+	// the pidfile, add an ignore mask. Same for passwd in case non-root
+	// crio containers
```

Comment:
This comment implies that `passwd` is only ignored for non-root crio containers. If I understand the code correct it ignores it for every configuration. So maybe change the wording.